zou/app/blueprints/crud/task_type.py

```diff
@@ -48,3 +54,28 @@ def post_update(self, instance_dict):
 
     def post_delete(self, instance_dict):
         tasks_service.clear_task_type_cache(instance_dict["id"])
+
+
+class TaskTypeLinksResource(Resource):
+    @jwt_required
+    def post(self):
+        data = request.json
+        task_type_link = ProjectTaskTypeLink.query.filter_by(
+            project_id=data["projectId"],
+            task_type_id=data["taskTypeId"]
+        ).first()
+
+        if task_type_link is None:
+            task_type_link = ProjectTaskTypeLink(
+                project_id=data["projectId"],
+                task_type_id=data["taskTypeId"],
+            )
+
+        task_type_link.priority = int(data["priority"])
```

Comment:
Use `.update` method there instead of using the db object.

Reply:
`ProjectTaskTypeLink` doesnt inherit from the `BaseMixin`, so it doesn't have access to the `.update()` method. I can make it inherit from the `BaseMixin` if you want (it will add an `id` column too)

Reply:
Yes, we can inherit from the BaseMixin. I think this object is going to get more complex over time (we will need to add a status list to each item).

Reply:
It's done :)